src/app/editor/contextView.js

```diff
@@ -159,7 +168,14 @@ class ContextView {
       <span class=${css.referencesnb}>${references}</span>
       <i data-action='previous' class="fa fa-chevron-up ${css.jump}" aria-hidden="true" onclick=${jump}></i>
       <i data-action='next' class="fa fa-chevron-down ${css.jump}" aria-hidden="true" onclick=${jump}></i>
+        ${showGasEstimation()}
     </div>`
+
+    function showGasEstimation () {
+      var estimatedGas = self._api.contextualListener.gasEstimation(node) + ' gas'
+      if (node.name === 'FunctionDefinition') var el = yo`<div class=${css.gasEstimation}> <img class=${css.gasStationIcon} title='Estimated gas price' src='https://png.icons8.com/gas-station/win8/50/000000'> ${estimatedGas}</div>`
```

Comment:
is that possible to you use an icon from font-awesome ?

Reply:
@yann300 Unfortunately not :( Was searching there first, but it doesn't exist. I really wanted to use a gas station icon. [This one](https://ethgasstation.info/images/ETHGasStation.png) would be my favorite, but it's a logo of some project... Maybe @ryestew could help us create one? Or we go with this one for now and if/when fontawesome adds gas station icon, we switch. I saw people already opening issues and asking for this icon at fontawesome, but so far nothing yet...

Reply:
ok then we need to import it in the project (we can't use an external link). you could convert it to base64

Reply:
@yann300 Should I maybe rather download it?

Reply:
Yes it's fine

Reply:
@yann300 Is everything else ok otherwise?

Reply:
just one little add that could be nice to add (please check above)